Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/Beckn/Search.hs

```diff
@@ -237,6 +239,16 @@ handler merchant sReq = do
           let onlyFPWithDrivers = filter (\fp -> isJust (find (\dp -> dp.variant == fp.vehicleVariant) driverPool)) farePolicies
           searchReq <- buildSearchRequest sReq merchantId fromLocation toLocation result.distance result.duration specialLocationTag area
           estimates <- mapM (SHEst.buildEstimate searchReq.id sReq.pickupTime result.distance specialLocationTag) onlyFPWithDrivers
+          version <- asks (.version)
+          let searchPayload = createSearchPayload (getId searchReq.id) Nothing searchReq.createdAt
+          envt <- createEvent "" (getId merchantId) "Search" version.getDeploymentVersion "dynamic-driver-offer-app" (Just $ Search searchPayload)
```

Comment:
Make no sense to use empty string as `personId`. Using `Nothing` would be better